### PR TITLE
Minor fixes to make it fully compatible with CentOS 7

### PR DESCRIPTION
**Pull request by grom-it**
Please review and integrate our minor changes that make your great package fully usable for us and other CentOS 7 users.

Cheers
Robert

### CODE REVIEW

**Comment by kazuhisya** [comment]
Thanks PR.

Is it possible to remove the commit of c62728e?
I thught it is undesirable for Release tag to have possibilities to vary according to environment.
Probably it always becomes 1 in my environment, and it is in the value that is different in the other environment having `$BUILD_NUMBER` .

**Comment by grom-it** [comment]
Hi Kazuhisa,

tl;dr: the build number is necessary for continuous integration builds of this package.

yes, it is possible to remove the build number gathered from an environment variable but I'd prefer to negotiate and explain why it seems necessary for us:
We appreciate your work and yours is the best current nodejs rpm for RHEL distributions I could find, even better than the nodesource-rpm recommended by nodejs.org. But regardless we trust you and respect your work we want to use this rpm in the productive environment of our business. So we prefer to build the rpm on ourselves and host it in our own repositories, so we have at least a little bit of control over changes before releasing them into production. 
We do this by having a continuous build chain that builds a new rpm version for every commit in the source repository. So it can happen that a package is built multiple times even without a change to the version/release number being made. To ensure these packages being deployed and updated anyway they need to be enumerated which every continuous integration tool (e.g. Jenkins, Teamcity, ...) supports by counting up a "build number".
To make it as compatible and less invasive as possible we read this build number from the environment variable `BUILD_NUMBER` or default to 1 if not set. So it will have no impact in environments where this variable is not defined. 

If you read until here I hope you understand our intention and accept this change.

Cheers
Robert
Lead Developer
Immobilien Scout GmbH

**Comment by kazuhisya** [comment]
Thank you with a detailed description. I understood.
And it's seems OK.

If an update of future Release Number is required, I will increment this.
ex: `${BUILD_NUMBER:-2}` 

Thanks for your pull request!
